### PR TITLE
Add new tempate FastAPI

### PR DESCRIPTION
**Pull request by balexandermunoz**
### New

- [x] Template - New `.gitignore` template

## Details

More information in [FastAPI .gitignore](https://github.com/tiangolo/fastapi/blob/master/.gitignore)

### CODE REVIEW

File: templates/FastAPI.gitignore [hunk 1]
```diff
@@ -0,0 +1,22 @@
+.idea
```
Comment (konstrybakov) [inline templates/FastAPI.gitignore:1]: I think this isn't specific to FastAPI

File: templates/FastAPI.gitignore [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+.ipynb_checkpoints
```
Comment (konstrybakov) [inline templates/FastAPI.gitignore:2]: I think this isn't specific to FastAPI, it relates to Jupyter Notebooks, right?

File: templates/FastAPI.gitignore [hunk 3]
```diff
@@ -0,0 +1,22 @@
...
+.mypy_cache
+.vscode
```
Comment (konstrybakov) [inline templates/FastAPI.gitignore:4]: Not related to FastAPI

File: templates/FastAPI.gitignore [hunk 4]
```diff
@@ -0,0 +1,22 @@
...
+__pycache__
+.pytest_cache
+htmlcov
+dist
+site
```
Comment (konstrybakov) [inline templates/FastAPI.gitignore:9]: Not strictly related to FastAPI

File: templates/FastAPI.gitignore [hunk 5]
```diff
@@ -0,0 +1,22 @@
...
+.coverage
+coverage.xml
+.netlify
```
Comment (konstrybakov) [inline templates/FastAPI.gitignore:12]: same here

**Review by konstrybakov — CHANGES_REQUESTED** [review]
Hi @balexandermunoz,

First of all, thank you for your contribution to our project.

However, I noticed that your suggested .gitignore file seems to be a direct copy from the fastapi repository. While some of these ignored paths and files are relevant for FastAPI projects, not all of them are specifically related to the framework.

The purpose of this FastAPI.gitignore file should be to exclude files and directories that are specifically generated by the FastAPI framework and which we typically don't want to include in version control.

Here's a list of entries from your suggestion that aren't specific to FastAPI:

```
.idea
.ipynb_checkpoints
.vscode
dist
site
.coverage
coverage.xml
.netlify
log.txt
Pipfile.lock
env3.*
env
docs_build
site_build
venv
docs.zip
```

While these entries are often useful to ignore in many Python projects, they are not specifically generated by FastAPI.

I kindly ask you to revise your PR to only include entries that are specifically related to FastAPI.

Thank you for your understanding and your help in keeping our project clean and well-organized.

**Comment by balexandermunoz** [comment]
Hi, thanks for the feedback.
The PR has been updated!

**Review by konstrybakov — COMMENTED** [review]
Upon further review, it seems that everything aside from test.db and archive.zip is accounted for in the Python .gitignore file provided here:

https://www.toptal.com/developers/gitignore/api/python

Given this, I would like to understand the purpose of the test.db and archive.zip files. How do they pertain to FastAPI?

Additionally, does FastAPI generate any specific files that should not be committed to version control?

**Pull request closed without merging**